store/i18n.js

```diff
@@ -2,6 +2,7 @@ import IntlMessageFormat from 'intl-messageformat';
 import { LOCALE } from '@/config/cookies';
 import { get } from '@/utils/object';
 import en from '@/assets/translations/en-us.yaml';
+import zhHans from '@/assets/translations/zh-hans.yaml';
```

Comment:
There hasn't been a 2nd real language so it's possible there's bugs but you shouldn't need this.  The idea is only English is always loaded and gets built into the main app bundle, because it's the default and the fallback for missing keys.

Any others get dynamically loaded and added to the state when needed, by the `load` action.